ruby/occurrence_hash/lib/occurrence_hash.rb

```diff
@@ -0,0 +1,7 @@
+class CharOccurrenceChecker
+  def char_occurrence(incoming_data)
```

Comment:
1. What is the meaning of `incoming_data`?
2. What is the meaning of `output_hash`?
3. Method name is still the same.
4. I don't see an extra commit for this  change. Any reason?